resources/views/emails/pilot_passport/enroll.blade.php

```diff
@@ -0,0 +1,17 @@
+@extends('layouts.email')
+
+@section('content')
+<p>Dear {{ $pilot->full_name }},</p>
+
+<p>Thank you for enrolling in the vZTL Pilot Passport {{ $data->title }} Challenge! You can view program requirements and track your progress on
+    our website at <a href="https://www.ztlartcc.org/pilot_passport" alt="Website">https://www.ztlartcc.org/pilot_passport</a>.</p>
+
+<p>Please remember that in order to get credit for visiting a participating airfield, you should plan to land and remain on the ground for 
+    about 5 minutes. You can immediately verify that we logged your visit by refreshing your passport book on our website or by monitoring
+    your email (we will send a confirmation email with each qualifying visit).</p>
```

Comment:
Maybe including a note that it could be up to 5 minutes after it's been logged that they get the email?

Reply:
It shouldn't be... are you seeing something that would delay the email by 5 minutes?

Reply:
Just with email queuing and if a lot of emails are being sent at a particular time, it could take up to 5 minutes (maybe more if there's really a huge number of emails in the queue) for the email to be processed in that queue

Reply:
Good point - I wasn't thinking about the new email queuing. Updated.